### PR TITLE
Updated Artifactory version to 5.1.0

### PR DESCRIPTION
**Pull request by jainishshah17**

### CODE REVIEW

File: incubator/artifactory/templates/deployment.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
       - name: {{ template "fullname" . }}
-        image: "{{default "docker.bintray.io/jfrog/artifactory-oss" .Values.image}}:{{default "4.16.0" .Values.imageTag}}"
+        image: "{{default "docker.bintray.io/jfrog/artifactory-oss" .Values.image}}:{{default "5.1.0" .Values.imageTag}}"
```
Comment (mgoodness) [inline incubator/artifactory/templates/deployment.yaml]: Let's drop the `default` statements here and use the `image` & `imageTag` values explicitly.

File: incubator/artifactory/values.yaml [hunk 2]
```diff
@@ -8,7 +8,7 @@ Component: "Artifactory"
 
 ## Uncomment following line if you want to run Artifactory-Pro
 #image: "docker.bintray.io/jfrog/artifactory-pro"
```
Comment (mgoodness) [inline incubator/artifactory/values.yaml:10]: Since we're dropping the `default` statements, set the image to `docker.bintray.io/jfrog/artifactory-oss`.

**Review by mgoodness — CHANGES_REQUESTED** [review]
Let's be explicit about using the `image` & `image tag` values rather than depending on the `default` function.

**Comment by jainishshah17** [comment]
@mgoodness made changes per suggestion. Please check

**Comment by lachie83** [comment]
@mgoodness Can you please review again?

**Comment by jainishshah17** [comment]
@mgoodness @lachie83  any update?

**Review by mgoodness — APPROVED** [review]

**Comment by mgoodness** [comment]
@jainishshah17 Sorry about the delay. Ready to merge!

**Pull request merged**